### DISCUSSION TITLE
Cant Connect Krita to Runpod ComfyUI

### DISCUSSION THREAD

**Discussion by youngdevil11** [Acly/krita-ai-diffusion#1036] [Q&A]
I am Getting this SSL error in Krita when I tried to connect runpod with Krita. I Tried to install new certificate also but it didn't worked. 

Error: Could not establish websocket connection at wss://t6nol7tlpr0jnb-3001.proxy.runpod.net: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:997)

**Comment by Nechaken** [comment]
Experiencing the same issue.  MacOS Sonoma with the 1.21.0 plugin.  Works fine from Windows to the same runpod address.

**Comment by Nechaken** [comment]
This works to get it started:

CERT_PATH=$(python3 -m certifi)
export SSL_CERT_FILE=${CERT_PATH}
export REQUESTS_CA_BUNDLE=${CERT_PATH}

cd /Applications/krita.app
./krita

**Reply by Acly** [reply]
I thought this was no longer needed after #841, I wonder if something changed...

**Reply by trung1-dothanh** [reply]
krita-ai-diffusion 1.25.0 on Fedora 40 can't work without this workaround. Thank you.
